observations/observation_realtime_streaming.py

```diff
@@ -0,0 +1,1108 @@
+      return arr
+except Exception as e:
+  errorLogger.error(e, exc_info=True)
+
+try:
+  def obj_creation(obSub):
+    # Debug log for survey submission ID
+    infoLogger.info(f"Started to process kafka event for the observation Submission Id : {obSub['_id']}. For Observation Question report")
+    observation_submission_id =  str(obSub['_id'])  
+    if check_observation_submission_id_existance(observation_submission_id,"observationSubmissionId","sl-observation"):
+      infoLogger.info(f"No data duplection for the Submission ID : {observation_submission_id} in sl-observation ")  
+      if obSub['status'] == 'completed': 
+        if 'isAPrivateProgram' in obSub :
+          completedDate = None
+          try:
+            completedDate = obSub['completedDate']
+          except KeyError:
+            pass
+          createdAt = obSub['createdAt']
+          updatedAt = obSub['updatedAt']
+          evidencesArr = [ v for v in obSub['evidences'].values() ]
+          evidence_sub_count = 0
+          entityId = obSub['entityId']
+
+          userSubType = None
+          try :
+            if 'userRoleInformation' in obSub:
+              userSubType = obSub["userRoleInformation"]["role"]
+          except KeyError:
+            userSubType = ''
+
+          rootOrgId = None
+          boardName = None
+          user_type = None
+          try:
+              if obSub["userProfile"] :
+                  if "rootOrgId" in obSub["userProfile"] and obSub["userProfile"]["rootOrgId"]:
+                      rootOrgId = obSub["userProfile"]["rootOrgId"]
+                  if "framework" in obSub["userProfile"] and obSub["userProfile"]["framework"]:
+                    if "board" in obSub["userProfile"]["framework"] and len(obSub["userProfile"]["framework"]["board"]) > 0:
+                      boardName = ",".join(obSub["userProfile"]["framework"]["board"])
+                  try:
+                    temp_userType = set([types["type"] for types in obSub["userProfile"]["profileUserTypes"]])
+                    user_type = ", ".join(temp_userType)
+                  except KeyError:
+                    pass
+
+          except KeyError :
+              pass
+          obsAppName = None
+          try :
+            obsAppName = obSub["appInformation"]["appName"].lower()
+          except KeyError :
+            obsAppName = ''
+          userRolesArrUnique = []
+          roleObj = {}
+          roleObj["role_title"] = userSubType
+          roleObj["user_boardName"] = boardName
+          roleObj["user_type"] = user_type
+          #commented it beacause "userLocation" key not present in userProfile
+          # if "userProfile" in obSub and len(obSub["userProfile"]["userLocations"])>0:
+          #  for ent in obSub["userProfile"]["userLocations"]:
+          #     roleObj["user_"+ent["type"]+"Name"] = ent["name"]
+          #     roleObj[ent["type"]+"_externalId"] = ent["id"]
+          #     roleObj[ent["type"]+"_code"] = ent["code"]
```

Comment:
@prashanthShiksha Not satisfied with the reason to comment this code out. Please get more clarity and let me know why was this removed